dds_cli/__main__.py

```diff
@@ -91,6 +91,20 @@
     highlight=False,
 )
 
+try:
+    with dds_cli.motd_manager.MotdManager() as getter:
+        motd = getter.get_latest_motd()
+        # message = motd.get("message")
+    dds_cli.utils.stderr_console.print("[bold]Important information:", motd.get("message"), "\n")
+except (
+    dds_cli.exceptions.AuthenticationError,
+    dds_cli.exceptions.ApiResponseError,
+    dds_cli.exceptions.ApiRequestError,
+    dds_cli.exceptions.DDSCLIException,
+) as err:
+    LOG.error(err)
+    sys.exit(1)
+
```

Comment:
I think we should only have: 
```
motd = dds_cli.motd_manager.MotdManager.get_latest_motd()
```
And then you can structure that function as the `get_user_name_if_logged_in` - just use `try except` without any specific exceptions. 
Then do
```
if motd:
    dds_cli.utils.stderr_console.print("[bold]Important information:", motd.get("message"), "\n")
```

So: 
```suggestion
motd = dds_cli.motd_manager.MotdManager.get_latest_motd()
if motd:
    dds_cli.utils.stderr_console.print(f"[bold]Important information: {motd} \n")
```